src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const total = days * 40;
+  let rentalCost;
+
+  if (days >= 1 && days < 3) {
```

Comment:
You can simplify the condition by just checking if `days < 3`. There's no need to check if `days >= 1` since `days` should always be a positive number.